### ISSUE TITLE
More Clarification Needed

### ISSUE THREAD

**Issue by buzzwick** [MicrosoftDocs/navdevitpro-content-pr#2174]
I am wondering if there needs to be some further clarification. The doc instructs to export the language file, then "open the file with a text editor or in Microsoft Office Excel."  Then the picture illustrates a file that has been parsed into two Excel columns--something that takes some artful Excel importing, not just "open". Then in the Import section, we get only "Select the file, and then choose Open." It does not describe how or if the parsed Excel file needs to be re-assembled before importing.  A little documentation on the identity/role of each of the elements on the language line would also be helpful. 
[Enter feedback here]

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: bde3484d-cec0-101f-c0c4-09f8b0fa7500
* Version Independent ID: bd4ff161-3dbc-892b-a188-35350db0882c
* Content: [Importing and Exporting Multilanguage Files - Dynamics NAV](https://docs.microsoft.com/en-us/dynamics-nav/how-to--add-translated-strings-by-importing-and-exporting-multilanguage-files)
* Content Source: [dynamics-nav/How-to--Add-Translated-Strings-By-Importing-and-Exporting-Multilanguage-Files.md](https://github.com/MicrosoftDocs/navdevitpro-content-pr/blob/live/dynamics-nav/How-to--Add-Translated-Strings-By-Importing-and-Exporting-Multilanguage-Files.md)
* Product: **dynamics-nav-2018**
* GitHub Login: @jswymer
* Microsoft Alias: **jswymer**

**Comment by edupont04** [comment]
Hello David,
There is no magic involved in opening a .txt file with multilanguage captions into Excel. Just open the .txt file as a text file delimited by colon (: ).
Rather, the issue is that it only works for multilanguage files exported from C/SIDE, not from .al files.

**Comment by buzzwick** [comment]
Eva,
"Would but it were so simple." No magic perhaps, but still more than a little deceptive. If you save that colon-delimited Excel file, Excel replaces the colon with a tab, rendering the file unusable for import ("There must be a colon in the string"). So you cannot just use the edited Excel file as described. Many more steps are required to reassemble the correct format before saving. How about including that error message in the text, thus alerting people to a detail that is hidden by the example? Why not at least show the actual edited text in its correct final form, instead of the slightly misleading Excel example? How about using a straight text example, which would be more accurate? My point is that the doc does not actually either show or tell a developer what the developer actually needs to produce. Regards!

**Comment by edupont04** [comment]
Good morning!
Well, I and my colleagues did this on a daily basis for years, so maybe we did take it for granted - the trick is to not save the file as an Excel book. We just enabled filters, did our work, removed the filters again, and then saved as the original .txt file. But there is no requirement to use Excel, and the screenshot is just there to show that the exported file format can be used for something useful, such as bulk-editing.

**Comment by buzzwick** [comment]
Hi Eva,
Okay, it took many tests, but finally I hit on the correct sequence. If the file is saved out of Excel as a “CSV” file (there are many “TXT” formats; these do NOT work), then the comma is (I submit, magically) replaced in the file. Brilliant. I have learned a new trick—which is good, because I have a LOT of these to do (basically localizing our entire add-on). After this experience, I am even more convinced that simply including the text of that error message somewhere in the doc would save a lot of people a lot of trial and error. You have the Curse of Knowledge; we tyros do not!

All the best,
David
P.S.: I have to ask: have you seen the Disney animated film “Wall-E”?

**Comment by edupont04** [comment]
Hi David,

Yes, I loved Wall-E, but it made my manager at the time change her pronunciation of my name, which was somewhat problematic ;-)

Anyway, for editing these .txt files, I simply don't remember exactly what we did back then, and our internal documentation has long since evaporated. It might even be Excel that has changed how it works with these delimiters - I just know that you're the first to report problems after this article has been looking like this for more than 10 years. I do respect the problems you're struggling with, but there isn't much I can do to help, unfortunately.

**Comment by buzzwick** [comment]
Hi Eva,

Let me guess: your manager started pronouncing your name “Eeeevaaaaaaa”?

Regarding Excel:  I believe that Office365 changed the way Excel behaves when saving files that were imported from .txt. I have noticed that the rhythm has changed (this is an operation I do a lot), and at least one prompt has been eliminated.

But we did come up with an Excel format that allows me to import as colon-delimited, save as Excel, send to the client to put German translations in a blank column. Then I concatenate the original stem, colon, and German translation and copy-paste that by itself into the text file I need for import. With practice we might eliminate a step or two, but this seems like it will work fine.

Thank you for your suggestions—they led me to uncover what was going on, which enabled me to come up with a workable solution.

All the best,

David
